types/lhr/settings.d.ts

```diff
@@ -69,6 +77,8 @@ export type ScreenEmulationSettings = {
   gatherMode?: boolean | string;
   /** Flag indicating that the browser storage should not be reset for the audit. */
   disableStorageReset?: boolean;
+  /** Flag indicating which kinds of browser storage should be reset for the audit. */
+  clearStorageTypes?: StorageType[]
```

Comment:
nit
```suggestion
  clearStorageTypes?: StorageType[];
```

Reply:
I couldn't find a formatter in the project.  How should I format my patch?

Reply:
`yarn lint` should help, I believe `yarn lint --fix` will work as well

Reply:
I ran `yarn lint --fix` before committing and it didn't make this change

Reply:
imagine formatting files by hand in 2023 :D